documentation/vespa-quick-start.html

```diff
@@ -67,7 +67,7 @@
     <strong>Ensure the application is active:</strong><br>
     Wait for a valid <samp>200 OK</samp> response from:
 <pre>
-[root@vespa-container #/] curl -s --head http://localhost:8080/ApplicationStatus 
+[root@vespa-container #/] curl --head http://localhost:8080/ApplicationStatus 
```

Comment:
Why this change? -s hides the progress stuff ?

Reply:
sorry my bad